src/Files.App/Files.App.csproj

```diff
@@ -75,6 +75,7 @@
         <PackageReference Include="LibGit2Sharp" Version="0.30.0" />
         <PackageReference Include="LiteDB" Version="5.0.19" />
         <PackageReference Include="LiveChartsCore.SkiaSharpView.WinUI" Version="2.0.0-rc1.2" />
+        <PackageReference Include="MessageFormat" Version="7.1.0" />
```

Comment:
We’re removing third party package references.
But for now it’s fine, I’ll make our own one when I have time: AppLocalizationService and AppLocalizationExtensions classes.

Reply:
I have already created such a solution, but it is not yet optimally fast, MessageFormat is very fast and small. I'm working on improving it, then I might as well submit my solution to the Files community

Reply:
How large is the package?

Reply:
Which one do you mean? MessageFormat is like this https://github.com/jeffijoe/messageformat.net/tree/master?tab=readme-ov-file#features , I don't know mine yet, I'm working on it

Reply:
MessageFormat